default/python/AI/ProductionAI.py

```diff
@@ -148,6 +149,11 @@ def _first_turn_action():
             fo.updateProductionQueue()
 
 
+def get_building_allocations() -> float:
```

Comment:
I guess this should also be named _get_building_allocations.
Also I wonder why some helper functions are placed before generate_production_orders and others below.
I intentionally put translators_wanted on top, because it is actually used by another module and all others below.

Reply:
I'll rename it later, I probably will split this module to separate files when will see the pattern. 

An order does not matter if this function is used only inside other functions.

Reply:
Some of the functions are created manually and some are by extracting action.